be/src/olap/schema_change.cpp

```diff
@@ -194,6 +188,105 @@ ColumnMapping* RowBlockChanger::get_mutable_column_mapping(size_t column_index)
         break; \
     }
 
+
+bool to_bitmap(RowCursor* read_helper, RowCursor* write_helper, const TabletColumn& ref_column,
+               int field_idx, int ref_field_idx, MemPool* mem_pool) {
+    write_helper->set_not_null(ref_field_idx);
+    BitmapValue bitmap;
+    if (!read_helper->is_null(ref_field_idx)) {
+        uint64_t origin_value;
+        char *src = read_helper->cell_ptr(ref_field_idx);
+        switch (ref_column.type()) {
+            case OLAP_FIELD_TYPE_TINYINT:
+                if (*(int8_t *) src < 0) {
+                    LOG(WARNING) << "The input: " << *(int8_t *) src
```

Comment:
How about return Status with this error message?
And do not print log here, it may lead to lots of log, and this error info should return to user.

Reply:
This is a good suggestion，but it involves a lot of modification points, I will add this feature optimization in the next PR

Reply:
> How about return Status with this error message?
> And do not print log here, it may lead to lots of log, and this error info should return to user.

+1